shared/profile/search/bar.js

```diff
@@ -5,38 +5,43 @@ import * as Styles from '../../styles'
 import flags from '../../util/feature-flags'
 import Search from './container'
 
-type Props = {||}
+type Props = {|onSearch: () => void|}
 type State = {|show: boolean|}
 
 class ProfileSearch extends React.PureComponent<Props, State> {
   state = {show: false}
 
   _ref = React.createRef()
-  _onShow = () => this.setState({show: true})
+  _onShow = () => {
+    this.setState({show: true})
+    this.props.onSearch()
+  }
   _onHide = () => this.setState({show: false})
   _getAttachmentRef = () => this._ref.current
   render() {
     return (
-      <Kb.Box2 alignSelf="flex-end" direction="vertical" ref={this._ref}>
+      <Kb.Box2 style={styles.container} direction="horizontal" ref={this._ref}>
         <Kb.ClickableBox
           onClick={this._onShow}
           style={Styles.collapseStyles([styles.searchContainer, this.state.show && {opacity: 0}])}
         >
-          <Kb.Icon
-            color={Styles.globalColors.black_50}
-            fontSize={Styles.isMobile ? 20 : 16}
-            style={styles.searchIcon}
-            type="iconfont-search"
-          />
-          <Kb.Text style={styles.searchText} type="BodySemibold">
-            Search people
-          </Kb.Text>
+          <Kb.Box2 direction="horizontal">
+            <Kb.Icon
+              color={Styles.globalColors.black_50}
```

Comment:
A little hard to read on profile headers
<img width="257" alt="image" src="https://user-images.githubusercontent.com/11968340/54388940-56081900-4675-11e9-8c64-00948b2ac0db.png">
admin build:
<img width="257" alt="image" src="https://user-images.githubusercontent.com/11968340/54388954-5dc7bd80-4675-11e9-93ad-5d4077267b28.png">

Reply:
lets leave this for a bit. i'm sure we'll take a visual pass and adjust this stuff. currently this is used in the followers/following section and the proof section on profiles so it gets a whit eback ground and a color background so in order to fix that better we'd have to add more plumbing